memstore/ingestion.go

```diff
@@ -206,6 +206,9 @@ func (shard *TableShard) insertPrimaryKeys(primaryKeyColumns []int, eventTimeCol
 				return nil, nil, nil, utils.StackError(err, "Event time for row %d is null", row)
 			}
 			primaryKeyEventTime = upsertBatch.ArrivalTime
+			if primaryKeyEventTime < shard.LiveStore.ArchivingCutoffHighWatermark {
```

Comment:
add comments saying that those missing event time records will not go to backfill queue if they are late

Reply:
done